chapters/crud.md

```diff
@@ -66,3 +66,16 @@ You can see this in the code listing in the beginning of this chapter.
    
 The update and delete operations use a masking underneath. This is done to maintain high performance overall. However, this means that old data will not be deleted, but will simply be masked as being deleted. Thus the size on disk (and in memory) of the indexer will grow over time if you perform update or delete operations. We recommend setting the `delete_on_dump` parameter of the `NumpyIndexer` to `True`. When the Flow is shut down, the data that has been marked as deleted will be permanently deleted before being saved to disk. By default the parameter is set to `False`, as setting it to `True` will make the shut down process slower.  
 
+
+6. **Compound Indexer**
+
+While using a `!CompoundIndexer`, it is not possible to return embeddings by virtue of its design. By definition, the [!CompoundIndexer](https://github.com/jina-ai/jina/blob/master/jina/resources/executors.requests.CompoundIndexer.yml) has an `!ExcludeQL` for all the embeddings:
```

Comment:
Not sure I understand what 'return embeddings' means

Reply:
@theUnkownName by `return` do we mean that Vector + KV indexer (Compound) won't persist the embedding?

Reply:
I think I got it. You meant return to the next Driver in the Flow? Yes, that would be the case with the current Exclude

Reply:
Maybe specify that in more detail

Reply:
It is what we discussed yesterday on the daily, that because of the ExcludeQL is not possible to keep the embeddings